plugins/services/src/js/containers/service-connection/__tests__/ServiceConnectionEndpointList-test.tsx

```diff
@@ -16,11 +16,14 @@ describe("ServiceConnectionEndpointList", () => {
 
       const endpointsTable = instance.find(".configuration-map-section");
 
-      expect(endpointsTable.length).toEqual(1);
+      expect(endpointsTable.length).toEqual(2);
 
       const endpointRows = instance.find(".configuration-map-row");
 
-      expect(endpointRows.length).toEqual(4);
+      expect(endpointRows.length).toEqual(8);
+
+      expect(instance.find('.configuration-map-value').at(1).text()).toEqual("8082")
+      expect(instance.find('.configuration-map-value').at(6).text()).toEqual("8443")
```

Comment:
i think it's ok to merge this but i'm not a fan of this kind of assertion at all.

whenever we change the fixture we likely need to update the whole file with new magic values.

it would be awesome to have some higher level selection function that goes like: 

"there's a row with key `Endpoint` and value `8443`" instead of "find all `.configuration-map-value`s, now give me the sixth, and now the text-node within that should be 8443".